src/Core/src/Handlers/Label/LabelHandler.Windows.cs

```diff
@@ -18,6 +18,13 @@ public static void MapBackground(LabelHandler handler, ILabel label)
 			handler.GetWrappedNativeView()?.UpdateBackground(label);
 		}
 
+		public static void MapOpacity(LabelHandler handler, ILabel label)
+		{
```

Comment:
Are you going to include the clipping logic for Windows in this PR?

Reply:
Windows already clips by default.